requirements.txt

```diff
@@ -15,3 +15,4 @@ lmdb==0.87
 h5py>=2.2.1,<=2.6.0
 pydot==1.0.28
 psutil>=1.2.1,<=3.4.2
+matplotlib==1.5.1
```

Comment:
Does this not work with 1.3.1 - the version that comes on Ubuntu 14.04?
http://packages.ubuntu.com/trusty/python-matplotlib

I really don't want to figure out how to backport the 16.04 package. Ideally, we would allow this range:

    matplotlib>=1.3.1,<=1.5.1